app/twitter_client.py

```
@@ -0,0 +1,60 @@
+
+    def post_tweet(self, text: str) -> Dict[str, Any]:
+        """Post a tweet to Twitter or simulate the request in dry-run mode."""
+
+        if not text:
+            raise ValueError("Tweet text must not be empty")
+
+        if self.dry_run:
+            return {"id": "dry-run", "text": text}
+
+        if not self.bearer_token:
+            raise RuntimeError("Twitter bearer token is required for posting tweets")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)